src/probeinterface/io.py

```diff
@@ -560,11 +560,25 @@ def read_3brain(file: Union[str, Path], mea_pitch: float = 42, electrode_width:
 
     h5py = import_safely("h5py")
     rf = h5py.File(file, "r")
+    if "3BRecInfo" in rf.keys():  # brw v3.x
+        # get channel positions
+        channels = rf["3BRecInfo/3BMeaStreams/Raw/Chs"][:]
+        rows = channels["Row"] - 1
+        cols = channels["Col"] - 1
+    else:  # brw v4.x
+        for key in rf:
+            if key[:5] == "Well_":
```

Comment:
Yes, I think it would make sense, the current implementation only supports the first well found ;) As I cannot test it, I did not implement it for multi-wells.

Reply:
Added a note about this in the docstring